### DISCUSSION TITLE
unable to find valid certification path to requested target

### DISCUSSION THREAD

**Discussion by chaithanya7818** [dbeaver/dbeaver#20689] [Q&A] [answered]
### Description

Hi,
I am using Dbeaver on Non Persistent VDI. While using NPVDI and trying to connect redshift clusters from Dbeaver through  OKTA, then I'm getting the error.
Please refer the attached screenshot for more details.

![error](https://github.com/dbeaver/dbeaver/assets/140507136/5833d282-b90a-4506-bdd4-1723f3f6d284)

### DBeaver Version

22.0.0.202203131528

### Operating System

windows

### Database and driver

Redshift database and JDBC driver with version jdbc42-2.0.0.6

**Comment by E1izabeth** [comment, marked as answer]
Could you try unchecking `Use Windows trust store` setting in `Preferences->Connections` and restart DBeaver?
![image](https://user-images.githubusercontent.com/28875055/201152880-ed80eb8f-a4da-411d-81c2-327509c216e9.png)
